assets/stylesheets/responsive/custom.scss

```diff
@@ -171,57 +171,94 @@ input.use-datepicker[type=text] {
   background:image-url('calendar.png') no-repeat 100px 5px $form_input-bg;
 }
 
-form input[type=submit], a.link_button_green, a.link_button_green_large {
-  text-decoration: none;
-  color: $button-color;
-  background-color: darken($button-bg, 5%);
-  display: inline-block;
-  cursor: pointer;
-  /* This style is expressed for all borders to avoid triggering an IE9 border rendering bug */
-  border-style: solid; border-color: darken($button-bg, 15%); border-width: 0 0 4px 0;
-  border-radius: 0;
-  padding: 3px 7px;
-  line-height: normal;
-  font-size: 1.1rem;
-  /* Same height as form elements */
-  height: 2.3125rem;
-  @include ie8 {
-     height: 2.3125em;
-     font-size: 1.1em;
-   }
-  vertical-align: top;
-  &:focus, &:hover{
+/* Button styles */
+.button-base {
+    padding: 0.5em 1.25em;
+    border-radius: 3px;
+    display: inline-block;
+    font-weight: 600;
     text-decoration: none;
-    background-color: darken($button-bg, 10%);
-    border-bottom: 4px solid darken($button-bg, 20%);
-  }
+    transition: background-color 300ms ease-out;
+    margin-bottom: 0;
+    font-family: $font-family;
+    &:hover,
+    &:active,
+    &:focus {
+        text-decoration: none;
+        transition: background-color 300ms ease-out;
+    }
 }
 
-@include lte-ie7{
+a.button,
+.button,
+.button:visited {
+    @extend .button-base;
+    color: #fff;
+    background-color: $button-bg;
+    border: none;
+    &:hover,
+    &:active,
+    &:focus,
+    &:visited:hover,
+    &:visited:active,
+    &:visited:focus {
+        background-color: darken($button-bg, 10%);
```

Comment:
Do you need to pass the `$button-bg` to `darken`? Is there a way to inherit that from `.button`, so in effect we just have something like `background-color: darken(10%)`?